src/Nethermind/Nethermind.Serialization.Rlp/HeaderDecoder.cs

```diff
@@ -75,6 +76,12 @@ public class HeaderDecoder : IRlpValueDecoder<BlockHeader>, IRlpStreamDecoder<Bl
                 blockHeader.BaseFeePerGas = decoderContext.DecodeUInt256();
             }
 
+            if (decoderContext.ReadNumberOfItemsRemaining() > 0 &&
```

Comment:
Why did you decide to not use WithdrawalTimestamp here like in BlockDecoder?

Reply:
With that, we're failing hive tests. They require this not to be timestamp-dependent, otherwise, it affects block hash computation.

Reply:
I see, thanks for the explanation, I will do the same with new 4844 field.

Reply:
No need in WithdrawalTimestamp in that decoder looks like